packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/ClusterManagersController.m

```diff
@@ -0,0 +1,157 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "ClusterManagersController.h"
+#import "FLTGoogleMapJSONConversions.h"
+
+@interface FLTClusterManagersController ()
+
+@property(strong, nonatomic) NSMutableDictionary *clusterManagerIdToManagers;
+@property(strong, nonatomic) FlutterMethodChannel *methodChannel;
+@property(weak, nonatomic) GMSMapView *mapView;
+
+@end
+
+@implementation FLTClusterManagersController
+- (instancetype)init:(FlutterMethodChannel *)methodChannel mapView:(GMSMapView *)mapView {
+  self = [super init];
+  if (self) {
+    _methodChannel = methodChannel;
+    _mapView = mapView;
+    _clusterManagerIdToManagers = [[NSMutableDictionary alloc] init];
+  }
+  return self;
+}
+
+- (void)addClusterManagers:(NSArray *)clusterManagersToAdd {
+  for (NSDictionary *clusterManager in clusterManagersToAdd) {
```

Comment:
The name `clusterManager` is the same as the `GMUClusterManager *clusterManager` below, can we use a different name to avoid confusion?

Reply:
Fixed.